pkg/apis/azuredisk/v1beta2/types.go

```diff
@@ -542,6 +542,8 @@ type AzDiskDriverConfiguration struct {
 	DriverName string `json:"driverName,omitempty"`
 	// The address to expose profiling data from the pprof package. If empty, no profiler server is started.
 	ProfilerAddress string `json:"profilerAddress,omitempty"`
+	// The duration for which events are retained by kube-apiserver
+	EventTTLInSec int `json:"eventTTLInSec,omitempty"`
```

Comment:
This should be part of the `ControllerConfiguration`.

Reply:
this is really more of kube-apiserver configuration & we allow the customer to tell us what they gave kube-apiserver b/c retrieving that ourselves wouldn't be easy, but I'll move it